src/network/mod.rs

```diff
@@ -130,12 +130,24 @@ impl ChatMessage {
 
 #[derive(Copy, Clone, Debug, ByteConvertable, PrototypeElement)]
 pub enum Sex {
-    Male,
     Female,
+    Male,
     Both,
     Server,
 }
 
+impl PartialEq for Sex {
```

Comment:
This can be derived on the struct by adding `PartialEq` to the line
```rust
#[derive(Copy, Clone, Debug, ByteConvertable, PrototypeElement)]
```
It's good practice to also derive `Eq`. So basically that line would become
```rust
#[derive(Copy, Clone, Debug, PartialEq, Eq, ByteConvertable, PrototypeElement)]
```
And this manual implementation can be removed completely